asterius/src/Asterius/Main.hs

```diff
@@ -406,9 +416,13 @@ ahcDistMain logger task (final_m, report) = do
     "[INFO] Writing JavaScript runtime modules to "
       <> show
         (outputDirectory task)
-  rts_files <- listDirectory $ dataDir </> "rts"
+  rts_files' <- listDirectory $ dataDir </> "rts"
+  let rts_files = filter (\x -> x /= "browser" && x /= "node") rts_files'
   for_ rts_files $
     \f -> copyFile (dataDir </> "rts" </> f) (outputDirectory task </> f)
+  let specificFolder = map toLower $ show $ target task
```

Comment:
The function mapping from a `Target` to the directory name would be nice to be factored out here. Indeed `map toLower . show` works here but good to avoid duplication a little bit, and IMO a plain pattern matching is better.

Reply:
OK, rolled back to pattern matching